Web/starter_project_barbies/components/about/SessionCard.tsx

```diff
@@ -0,0 +1,20 @@
+import { SessionProps } from '@/types/AboutTypes'
```

Comment:
the prop type definition should follow a pattern of `[ComponentName]Props` so this should be `SessionCardProps`